docs/guides/gutenberg-blocks.md

```diff
@@ -0,0 +1,81 @@
+# Developing Gutenberg Blocks for Jetpack
+
+_Note: Since the Gutenberg SDK is still being actively developed, the development workflow described here is subject to frequent change (as of October 2018). Notably, we're currently working on a different (lerna and npm based) deploy workflow. Be sure to check back frequently!_
+
+1.  Install & activate the [Gutenberg plugin](https://wordpress.org/plugins/gutenberg/).
+
+    If you use Jetpack-Docker, you can use WP-CLI:
+
+        yarn docker:wp plugin install gutenberg --activate
+
+1.  In Jetpack, enable loading block assets by adding following filters to your local mu-plugins folder:
+
+        add_filter( 'jetpack_gutenberg', '__return_true', 10 );
+        add_filter( 'jetpack_gutenberg_cdn', '__return_false', 10 );
+
+    If you use Jetpack-Docker, you could add these to `docker/mu-plugins/0-custom.php`
+
+    Setting these might be useful for debugging:
+
+        define( 'SCRIPT_DEBUG', true );
+        define( 'GUTENBERG_DEVELOPMENT_MODE', true );
+
+    If you use Jetpack-Docker, you could add these to `docker/wordpress/wp-config.php`
+
+    Those filters take action here: https://github.com/Automattic/jetpack/blob/b4a057fad975f3db8097fd62e702e276fd3d4389/class.jetpack.php#L7272-L7381
+
+1.  Jetpack will now load these files when editing posts in Gutenberg:
+
+        _inc/blocks/editor.css
+        _inc/blocks/editor.rtl.css
+        _inc/blocks/editor.js
+        _inc/blocks/view.css
+        _inc/blocks/view.rtl.css
+        _inc/blocks/view.js   
+
+    Without setting `jetpack_gutenberg_cdn` to false, Jetpack would load these assets from CDN with 24h cache buster:
+
+    https://s0.wp.com/wp-content/mu-plugins/jetpack/_inc/blocks/
+
+    This is great for letting team-outsiders test our current set of Jetpack blocks; let's deploy these when ever we have something to show.
+
+    There is a helper-script to produce these assets locally for your sandbox so that you can then commit+deploy: https://opengrok.a8c.com/source/xref/trunk/bin/calypso/build-a8c-gutenberg-blocks.sh 
```

Comment:
You can't leave that link here, it links to a private resource.

Reply:
D'oh, good spot, thanks!

Reply:
Hmm, we don't seem to have GitHub shorthands for OpenGrok (or files in Trac -- only revisions and tickets). Is it fine to refer to the file path here (i.e. without the protocol and hostname)?